cfpb/cfgov-refresh
# Standardize most parts of email sign-up forms across the site

This PR addresses https://GHE/CFGOV/platform/issues/3202, standardizing most parts of email sign-up forms across the site to eliminate complexity and the inconsistencies that result.

It does a lot of things, which are broadly outlined by the commit log. I am running dangerously short on time before I have to quit for the day, but I really wanted to get this up for review before I'm out for the rest of the week. If a more thorough description is needed, I can do that on Monday.


## Testing

1. Pull branch
1. Run migrations
1. `gulp styles`
1. Look around at various email sign-up forms and see that they are now consistent in their:
   - Field labels
   - Field type and required-ness
   - Field placeholder text
   - Submit button text
   - Privacy Act statement link text

Be sure to check the special implementations in the three Buying a House tools (e.g., Rate Explorer) and the email pop-up styling, as well. They were also modified to match the standard `EmailSignup` organism as closely as possible.


## Screenshots

![screen shot 2019-02-20 at 18 15 35](https://user-images.githubusercontent.com/1044670/53131731-90313f80-353b-11e9-91d2-4a30f224d87f.png)


## Checklist



## Testing checklist

### Browsers


### Accessibility


### Other
